src/App.tsx

```diff
@@ -22,18 +23,34 @@ function App(): JSX.Element {
     setIsPaused((prev) => !prev);
   };
 
+  const onCameraControl = (direction: Point) => {
```

Comment:
We usually name it like
```suggestion
  const handleCameraDirection = (direction: Point) => {
```